### PR TITLE
Some Sinta Clothes

### PR DESCRIPTION
**Pull request by Geevies**
* Added new clothes for unathi, wasteland kilts, straw hats, wasteland goggles, standalone hoods.
* Roughspun robes are now recolourable. Mogazali attire is no longer recolourable.
* Added footwraps and caligae to the xeno clothes closet.

I didn't get a picture of everything, since sprites can be reviewed in icondiffbot, but here's one of the roughspun robes.
image

Feedback Thread:
https://forums.aurorastation.org/topic/14961-dronz-big-lizard-clothes-thread

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by fernerr — APPROVED** [review]

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -230,7 +230,7 @@ BLIND     // can't see anything
 
 /obj/item/clothing/glasses/safety/goggles/wasteland
 	name = "wasteland goggles"
-	desc = "A common pair of goggles, usually worn by Unathi who travel the wasteland. It's invaluable for keeping radioactive sand out of eyes."
+	desc = "A pair of old goggles common in the wasteland. Often worn by sailors and fishermen on the sea to keep saltwater out of their eyes before the war, now they take on new life to keep sand out of the eyes."
```
Comment (Haydizzle) [inline code/modules/clothing/glasses/glasses.dm:233]: A lot of the items here look pretty good. Just looking at the descriptions to be a little less winded;

"A pair of old goggles common in the Wasteland. A few denizens unfortunate enough to not keep this protection on them after the nukes dropped no longer have the ability to see."

File: code/modules/clothing/suits/alien.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 /obj/item/clothing/suit/unathi/robe/kilt
 	name = "wasteland kilt"
-	desc = "A simple kilt made out of basic fabrics. Easy to make, easy to move in, this is the ideal casual wear for patrolling a nuclear wasteland."
+	desc = "A long tunic made of old banners and other available materials, it acts as a kilt for the poorest or most free-spirited of wastelanders, who aren't afraid to let the sun and sand strike their scales."
```
Comment (Haydizzle) [inline code/modules/clothing/suits/alien.dm]: "commoner's kilt"
"A long tunic made of old material that acts as a kilt for the poorest of unathi, who aren't afraid to let the sand and sun strike their scales."

File: code/modules/clothing/under/accessories/xeno/unathi.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/accessory/sinta_hood
-	name = "sinta hood"
-	desc = "A large padded hood, worn by Sinta to protect them from the elements."
+	name = "clan hood"
+	desc = "A hood worn commonly by travelers, explorers, and levies. It's main purpose is to block the glare of sun when traveling, and to represent their clan. Often they have the names sewn into the inside to help identify the owner should it, or the owner, be lost."
```
Comment (Haydizzle) [inline code/modules/clothing/under/accessories/xeno/unathi.dm:3]: "A hood worn commonly by unathi away from home. No better way of both representing your clan to foreigners and keeping the sun out of your eyes in style!"

**Review by Haydizzle — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 4]
```diff
@@ -71,6 +71,14 @@
 	sort_category = "Xenowear - Unathi"
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/accessory/sinta_surcoat
+	display_name = "ghazak"
```
Comment (Haydizzle) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm:75]: The design is quite literally the flag, and it looks somewhat tacky. I'm opposed to this being added in its current iteration. A better implementation of something like a shoulder pauldron or cape would work well for adding the colors without it being the centerpiece.

**Review by Haydizzle — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by Haydizzle — APPROVED** [review]

**Pull request merged**